R/vault_api_client.R

```diff
@@ -34,37 +32,72 @@
 ##'   # cleanup
 ##'   server$kill()
 ##' }
-NULL
-
-
 vault_api_client <- R6::R6Class(
   "vault_api_client",
   inherit = vault_client_object,
   cloneable = FALSE,
 
   public = list(
+    ##' @field addr The vault port
```

Comment:
The addr is the whole vault address incl port? e.g. "https://support.montagu.dide.ic.ac.uk:8200"

Reply:
oh dear, this is wrong - thanks